src/Preview/Preview.tsx

```diff
@@ -607,6 +610,7 @@ const Preview: React.FC<PreviewProps> = ({
             sx={{
               transform: `scale(${zoomPercentage / 100})`,
               opacity: isCurrentAssetReady ? 1 : 0,
+              transformOrigin: (zoomPercentage === zoomDefault || zoomPercentage === zoomToFitPercentage) ? 'center' : 'top left',
```

Comment:
I'm not sure what causes this issue if zooming, seems everything is on the right side when under versionComparison

Should we just disable zooming when isVersionComparison=true?

@jcmcardama Please ask UIUX - looks weird for endusers
<img width="1792" alt="Screenshot 2024-09-25 at 7 29 30 PM" src="https://github.com/user-attachments/assets/69fcd3b4-feda-4a27-97d2-7a4f16388f44">